Wsl.psm1

```diff
@@ -272,6 +272,34 @@ function Get-WslDistribution
     }
 }
 
+<#
+.EXTERNALHELP
+Wsl-help.xml
+#>
+# Retrieves listing of distributions available from online sources
+function Get-WslDistributionOnline {
+    [CmdletBinding()]
+    param(
+    )
+
+    process {
+        $store = $false
+        Invoke-Wsl "--list", "--online" -IgnoreErrors | Select-Object -Skip 1 | ForEach-Object {
+            $name, $friendlyName = $_ -split ' ', 2
+            if ($store -eq $true ) {
+                $friendlyName = $($friendlyName.Split(@(" "), [System.StringSplitOptions]::RemoveEmptyEntries)) -join " "
+                [PSCustomObject]@{

```

Comment:
Please create a custom class at the top of the file and return that instead, to be consistent with other functions that return data in this module. Make sure it matches the name of the Output type shown in the external help file you added.

Reply:
Updated.